### ISSUE TITLE
Universal declaration of human rights corpus, non-truncated

### ISSUE THREAD

**Issue by alexrudnick** [nltk/nltk#84]
The UDHR corpus is available from the following URL, in plain text and XML
formats.  This would be good to have as a &quot;udhr2&quot; corpus.  (We can't
replace the existing one as its discussed in the book.)

http://unicode.org/udhr/downloads.html

<p>Migrated from http://code.google.com/p/nltk/issues/detail?id=448</p>

**Comment by kmike** [comment]
+1. Existing UDHR corpus is broken in many ways.

**Comment by heatherleaf** [comment]
Steven has said that it's ok to break consistency with the book, in the new NLTK 3. So why not replace `udhr` completely in the `2and3` branch?

**Comment by kmike** [comment]
I don't think it is possible because the data in https://github.com/nltk/nltk_data is not related to NLTK releases, and replacing udhr in nltk_data would break old releases. We may rename the corpus reader, but 'udhr' will remain old 'udhr' in nltk.download and this would be confusing. We may develop a versioning in nltk_data, but I think the resulting complexity doesn't worth it, and udhr2 is fine.

**Comment by stevenbird** [comment]
That's a good point, and udhr2 would be fine.

**Comment by stevenbird** [comment]
I've added udhr2 to nltk_data. I've removed udhr_ from the filenames, and removed the 6-line header from each file.

**Comment by kmike** [comment]
There is still no reader for that corpus. As versioning is now implemented, it may be better to use it for UDHR before NLTK3 release to check how it works in practice.

**Comment by stevenbird** [comment]
I've added a corpus reader for udhr2 in https://github.com/nltk/nltk/commit/e09cd2f4a62540ced4b26f20a26114681851d463

Note that we do not yet have support for corpus versioning correlated to versions of NLTK. We have support for versions correlated to Python 2 vs 3. And we trivially have support for adding new corpora whose names are based on existing corpora. We have worked out how to do the versioning that is sensitive to versions of NLTK but there has not yet been a need for this.
